gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -128,15 +115,30 @@ fluid.defaults("gpii.flowManager.local", {
             }
         }
         */
-        // TODO: After FLUID-6148 is implemented, directly saving "defaultLifecycleInstructions" into options should
-        // be replaced by expressing {that}.defaultLifecycleInstructions as an expander in the component's options
-        "userLogonHandling.defaultLifecycleInstructions": {
-            record: "{that}.options.defaultLifecycleInstructions",
-            target: "{that gpii.lifecycleManager.userLogonHandling.stateChangeHandler}.options.defaultLifecycleInstructions"
+        defaultSettingsDataPromise: "@expand:fluid.promise()"
```

Comment:
This should be in members rather than options since although it doesn't receive direct modifications, it is morally mutable since it has methods and state

Reply:
This definition is already in the "members" block.